Sources/OktaIdx/Extensions/InteractionCodeFlow+DeviceIdentifier.swift

```diff
@@ -0,0 +1,82 @@
+//
```

Comment:
Is this filename the standard naming convention for extensions in swift?

Reply:
It is. Filenames in Swift Packages have to be unique, regardless of which directory they're in, so the `+` is a common way to indicate an extension to another class or struct.

Reply:
Ah! Good to know. Haven't seen this pattern in android land before